odlcuda/cu_ntuples.py

```diff
@@ -195,494 +322,265 @@ def element(self, inp=None, data_ptr=None):
             else:
                 raise ValueError('cannot provide both `inp` and `data_ptr`')
 
-    @property
-    def element_type(self):
-        """ `CudaNtuplesVector` """
-        return CudaNtuplesVector
+    def _lincomb(self, a, x1, b, x2, out):
+        """Linear combination of ``x1`` and ``x2``, assigned to ``out``.
 
-    impl = 'cuda'
+        Calculate ``z = a * x + b * y`` using optimized CUDA routines.
 
-    @staticmethod
-    def available_dtypes():
-        """Return the available data types."""
-        return CUDA_DTYPES
+        Parameters
+        ----------
+        a, b : `LinearSpace.field` `element`
```

Comment:
Is \`field.element\` not ok here?

Reply:
No idea. Docs are not built anyway, so I can write anything.